### PR TITLE
Clean up the Help Page

### PR DESCRIPTION
**Pull request by xuhang57**
Delete some contents that we don't need in Help Page.

### CODE REVIEW

File: troposphere/static/js/components/help/HelpPage.jsx [hunk 1]
```diff
@@ -36,29 +14,13 @@ export default React.createClass({
             return <div className="loading"></div>;
         }
 
-        var resourceElements = resources.map(function(resource) {
```
Comment (kamfonik) [inline troposphere/static/js/components/help/HelpPage.jsx:39]: This is declared on line 33 above, can it be removed there too?

**Review by kamfonik — APPROVED** [review]
One small note, otherwise looks good

File: troposphere/static/js/components/help/HelpPage.jsx (hunk 1 above)
Comment (lucashxu) [reply troposphere/static/js/components/help/HelpPage.jsx:39]: good catch. Thanks!

**Review by rob-baron — APPROVED** [review]
ok.

**Pull request merged**